Samples/Samples/ViewModel/HomeViewModel.cs

```diff
@@ -204,6 +204,12 @@ public HomeViewModel()
                     typeof(VibrationPage),
                     "Quickly and easily make the device vibrate.",
                     new[] { "vibration", "vibrate", "hardware", "device" }),
+                new SampleItem(
+                    "📳",
+                    "HapticFeedback",
+                    typeof(HapticFeedbackPage),
+                    "HapticFeedback",
+                    new[] { "HapticFeedback", "Haptic", "Feedback", "hardware", "device" }),
```

Comment:
```suggestion
                    new[] { "haptic", "feedback", "hardware", "device" }),
```

I think this is case sensitive...